locales/en-US/server.ftl

```diff
@@ -0,0 +1,161 @@
+// global phrases shared across pages, prefixed with 'g'
+
+// TODO: replace each page's reference to this with the global
+gMyShots = My Shots
+gHomeLink = Home
+gNoShots
+  .alt = no Shots found
+
+// creating page
+
+// Localization note: {title} is a placeholder for the title of the web page
+// captured in the screenshot. The default, for pages without titles, is
+// creatingPageTitleDefault.
+creatingPageTitle = Creating {title}
+creatingPageTitleDefault = page
+
+// home page
+
+homePageDescription
+  .content = Intuitive screenshots baked right into the browser. Capture, save and share screenshots as you browse the Web using Firefox.
+homePageButtonMyShots = Go To My Shots
+homePageTeaser = Coming Soon...
+
+// leave screenshots page
+
+leavePageConfirmDelete = Confirm account deletion
+leavePageErrorAddonRequired = You must have the addon installed to delete your account
+// TODO: do we need this separate error message?
+leavePageErrorAddonRequired2 = You must have the addon installed to leave
+leavePageErrorGeneric = An error occurred
+leavePageWarning = This will permanently erase all of your Firefox Screenshots data.
+leavePageButtonProceed = Proceed
+leavePageButtonCancel = cancel
+leavePageDeleted = All of your screenshots have been erased!
+
+// not found page
+
+notFoundPageTitle = Page Not Found
+notFoundPageDescription = The page was not found.
+notFoundPageMyShotsLink = My Shots
+
+// shot page
+
+// In the following error messages, {status} is a placeholder for an HTTP status
+// code, like '500'. {statusText} is a text description of the status code, like
+// 'Internal server error'.
+shotPageAlertErrorUpdatingExpirationTime = Error saving expiration: {status} {statusText}
+shotPageAlertErrorDeletingShot = Error deleting shot: {status} {statusText}
+shotPageAlertErrorUpdatingTitle = Error saving title: {status} {statusText}
+
+shotPageShareButton
+  .title = Share
+shotPageCopy = Copy
+shotPageCopied = Copied
+shotPageShareFacebook
+  .title = Share to Facebook wall or message
+// TODO: awkward phrase, maybe 'Share on Twitter'?
+shotPageShareTwitter
+  .title = Share to a tweet
+shotPageSharePinterest
+  .title = Share to Pinterest
+shotPageShareEmail
+  .title = Create email with link
+shotPageShareLink = Get a shareable link to this shot:
+shotPagePrivacyMessage = This shot is only visible to you until you share the link.
+shotPageCopyImageText
+  .label = Copy Image Text
+shotPageConfirmDeletion = Are you sure you want to delete this shot permanently?
+// Note: {timediff} is a placeholder for a future time like 'tomorrow' or 'next week'
+// TODO: "in {timediff}" probably won't work across languages. maybe an absolute date is best.
+shotPageExpirationMessage = If you do nothing, this shot will be permanently deleted in {timediff}.
+// Note: {timediff} is a placeholder for a time duration phrase like '30 days'
+// TODO: again, "for {timediff}" may not work well.
+shotPageRestoreButton = restore for {timediff}
```

Comment:
(pinged wrong GH username) For a moment I forgot we were talking about the website… Since it's using Fluent, there's proper plural support, so we should be able to localize this.
http://projectfluent.io/fluent/guide/selectors.html